ProtoGame2D/Code/Game/Code/Paddle.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Paddle.h"
+
+#include "Engine/Core/CoreIncludes.h"
+#include "Engine/Graphics/Renderer.h"
+#include "Engine/Graphics/Shader.h"
+
+Paddle* g_Paddle = nullptr;
+
+Paddle::Paddle()
+{
+}
+
+Paddle::~Paddle()
+{
+}
+
+void Paddle::Update(float deltaseconds)
+{
+	UNUSED(deltaseconds)
+}
+
+void Paddle::Render(Shader shader)
+{
+	Renderer::GetInstance()->DrawQuad(Vec2(200.0f, 200.0f), Vec2(100.0f, 50.0f), Vec4(1.0f, 0.0f, 1.0f, 1.0f), "../Data/Textures/stripes.png" , shader);
```

Comment:
Hardcoded values unacceptable. Create gamecommon.h and created constexpr variable for these. Preferably variable names in all Caps.